packages/web/components/Setup/SetupAvailability.tsx

```diff
@@ -65,7 +65,7 @@ export const SetupAvailability: React.FC<SetupAvailabilityProps> = ({
   return (
     <FlexContainer>
       <MetaHeading mb={5} textAlign="center">
-        Availability
+        Avail&#xAD;ability
```

Comment:
why?

Reply:
It's a soft hyphen that will break on displays that are too narrow.

One of the default mobile devices in Chrome will trigger it: iPhone 4, I think.

Reply:
Ah, interesting